src/records/a_record.rs

```diff
@@ -90,61 +90,59 @@ impl<'a> Manufacturer<'a> {
 
     pub fn to_single_char(&self) -> Option<u8> {
         use self::Manufacturer::*;
-
-        #[rustfmt::skip]
+        // It's sad that rustfmt currently nukes the alignment on these match arms
         match self {
-            Aircotec                  => Some(b'I'),
-            CambridgeAeroInstruments  => Some(b'C'),
-            DataSwan                  => Some(b'D'),
-            EwAvionics                => Some(b'E'),
-            Filser                    => Some(b'F'),
-            Flarm                     => Some(b'G'),
-            Garrecht                  => Some(b'A'),
-            ImiGlidingEquipment       => Some(b'M'),
-            LxNavigation              => Some(b'L'),
-            LxNav                     => Some(b'V'),
-            NewTechnologies           => Some(b'N'),
-            NielsenKellerman          => Some(b'K'),
-            Peschges                  => Some(b'P'),
-            PrintTechnik              => Some(b'R'),
-            Scheffel                  => Some(b'H'),
+            Aircotec => Some(b'I'),
+            CambridgeAeroInstruments => Some(b'C'),
+            DataSwan => Some(b'D'),
+            EwAvionics => Some(b'E'),
+            Filser => Some(b'F'),
+            Flarm => Some(b'G'),
+            Garrecht => Some(b'A'),
+            ImiGlidingEquipment => Some(b'M'),
+            LxNavigation => Some(b'L'),
+            LxNav => Some(b'V'),
+            NewTechnologies => Some(b'N'),
+            NielsenKellerman => Some(b'K'),
+            Peschges => Some(b'P'),
+            PrintTechnik => Some(b'R'),
+            Scheffel => Some(b'H'),
             StreamlineDataInstruments => Some(b'S'),
-            TriadisEngineering        => Some(b'T'),
-            Zander                    => Some(b'Z'),
-            UnknownSingle(s)          => Some(*s),
-            _                         => None,
+            TriadisEngineering => Some(b'T'),
+            Zander => Some(b'Z'),
+            UnknownSingle(s) => Some(*s),
+            _ => None,
```

Comment:
we could think about using https://docs.rs/strum/ for these enums and use `EnumProperty` to encode the one-letter and three-letter variants 🤔